noisemaker-hackathon/src/containers/Button/Button.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import './Button.css';
+const AudioContext = window.AudioContext
+const sound = new AudioContext()
+
+export class Button extends Component {
+  constructor() {
+    super()
+    this.audioNode = React.createRef();
+  }
+
+
+  playSound = () => {
+      this.audioNode.current.play()
+  }
+
+
+  render() {
+    return (
+      <section className="button-body">
+        <audio ref={this.audioNode} id="noise" src="/A-Tone.mp3"></audio>
```

Comment:
I do not know much about react apps but I could not get the sound to play, maybe it has to do with how you loaded the resource?

Reply:
ya i'm not sure about "/A-tone.mp3". I think it should be "./A-tone.mp3"  or just "A-tone.mp3". I also could not hear the tone.

Reply:
It broke when I deployed :-(.  Thanks for taking a look though!

Reply:
I also couldn't get the button to work.  I've struggled with audio in the past, and I think that one thing I'd try here is rather than pass the path to the audio file to src, require the audio file like const sound = require('/A-Tone.mp3') and the src={sound}

I can't swear that works but I think that is what I'd try.

Reply:
It's fixed!